config/manager/manager.yaml

```diff
@@ -57,37 +57,45 @@ spec:
       terminationGracePeriodSeconds: 10
 ---
 apiVersion: dm.cray.hpe.com/v1alpha1
-kind: RsyncTemplate
+kind: DataMovementManager
 metadata:
-  name: rsynctemplate
+  name: manager-controller-manager
   namespace: system
 spec:
   selector:
     matchLabels:
       cray.nnf.node: "true"
-  hostPath: "/mnt"
-  mountPath: "/mnt"
+  hostPath: "/mnt/nnf"
+  mountPath: "/mnt/nnf"
   template:
     spec:
+      serviceAccountName: nnf-dm-controller-manager # not sure why 'controller-manager' isn't patched to include the 'nnf-dm-' prefix
       tolerations:
       - key: "cray.nnf.node"
         operator: "Equal"
         value: "true"
         effect: "NoSchedule"
       shareProcessNamespace: true
       containers:
-        - name: rsync-controller
-          image: controller:latest
+        - name: worker
+          image: ghcr.io/nearnodeflash/nnf-mfu:latest
+          command: 
+            - /usr/sbin/sshd
+          args:
+            - -De
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SETUID", "SETGID", "MKNOD"]
+        - name: manager
           command: 
-            - /manager
+          - /manager
           args:
-            - --controller=node
+          - --leader-elect=false
+          - --controller=node
```

Comment:
This "controller=node" will be overridden in createDeploymentIfNecessary() to be "controller=default", so during a live run we have no cases that actually call main.go with "controller=node"... have I got that right?

Reply:
So datamovementmanager_controller.go creates both a Deployment and a DaemonSet.  The deployment runs "controller=default", and the DaemonSet runs "controller=node".